tests/importer/test_importer.py

```diff
@@ -10,6 +10,10 @@
 import ast
 import tempfile
 from fractions import Fraction
+import pytest
+
+
+PYTHONDONTWRITEBYTECODE = os.environ.get('PYTHONDONTWRITEBYTECODE')
```

Comment:
We only use this once, so we don't need the assignment.

Reply:
Fair enough